internal/config/defaults.go

```diff
@@ -10,13 +14,13 @@ var (
 
 	MeshSpecDefaults = map[string]string{
 		"name":    "Consul",
-		"status":  "not installed",
+		"status":  status.NotInstalled,
 		"version": "1.8.2",
 	}
 
 	MeshInstanceDefaults = map[string]string{
```

Comment:
Out of curiosity what prompts the change in these defaults and how do you override them? Are the defaults some stable versions of the mesh?

Reply:
Right now, this is "placeholder functionality" following the pattern in the Kuma adapter, and just informative, changing the version here would not have an effect (yet). This version was the latest version when I updated the adapter, and the  manifest is generated manually using the Helm chart.

New stable versions would prompt change, yes. 

As the config is based on Viper, its mechanisms would be used to override defaults.